src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -500,8 +501,8 @@ namespace mallocMC
                      */
                     alpaka::math::max(acc, wastefactor * bytes, +minChunkSize));
 
-                const uint32 startblock = _firstfreeblock;
-                const uint32 random_start_page = startpage + startblock * pagesperblock;
+                // global page index
+                const uint32 random_start_page = start_page_in_block + _firstfreeblock * pagesperblock;
```

Comment:
```suggestion
                const uint32 global_start_page = start_page_in_block + _firstfreeblock * pagesperblock;
```

I think global is better fitting in the name, since this function already use this kind of naming for some other variables. And the fact it is randomly choosen doesn't mattter for the rest of the code, it is only important that it is the fixed start page

Reply:
changed